wire/wire.go

```diff
@@ -0,0 +1,55 @@
+package wire
```

Comment:
Good idea to make this package here

Reply:
Should help ease the upgrade considerably :)